ethsigner/commandline/src/test/java/tech/pegasys/ethsigner/CommandlineParserTest.java

```diff
@@ -75,8 +75,16 @@ void fullyPopulatedCommandLineParsesIntoVariables() {
         .isEqualTo(new File("./keystore.pfx"));
     assertThat(config.getTlsOptions().get().getKeyStorePasswordFile())
         .isEqualTo(new File("./keystore.passwd"));
-    assertThat(config.getTlsOptions().get().getKnownClientsFile())
+    assertThat(config.getTlsOptions().get().getClientAuthConstraints().get().getKnownClientsFile())
```

Comment:
nit: extract to a variable can shorten call.

Reply:
👍